rfc/5280/index.js

```diff
@@ -0,0 +1,857 @@
+  try {
+  var asn1 = require('asn1.js');
+} catch (e) {
+  var asn1 = require('../..');
+}
+
+/**
+ * RFC5280 X509 and Extension Definitions
+ */
+
+var rfc5280 = exports;
+
+// OIDs
+var x509OIDs = {
+  '2 5 29 9': 'subjectDirectoryAttributes',
+  '2 5 29 14': 'subjectKeyIdentifier',
+  '2 5 29 15': 'keyUsage',
+  '2 5 29 17': 'subjectAlternativeName',
+  '2 5 29 18': 'issuerAlternativeName',
+  '2 5 29 19': 'basicConstraints',
+  '2 5 29 20': 'cRLNumber',
+  '2 5 29 21': 'reasonCode',
+  '2 5 29 24': 'invalidityDate',
+  '2 5 29 27': 'deltaCRLIndicator',
+  '2 5 29 28': 'issuingDistributionPoint',
+  '2 5 29 29': 'certificateIssuer',
+  '2 5 29 30': 'nameConstraints',
+  '2 5 29 31': 'cRLDistributionPoints',
+  '2 5 29 32': 'certificatePolicies',
+  '2 5 29 33': 'policyMappings',
+  '2 5 29 35': 'authorityKeyIdentifier',
+  '2 5 29 36': 'policyConstraints',
+  '2 5 29 37': 'extendedKeyUsage',
+  '2 5 29 46': 'freshestCRL',
+  '2 5 29 54': 'inhibitAnyPolicy',
+  '1 3 6 1 5 5 7 1 1': 'authorityInformationAccess',
+  '1 3 6 1 5 5 7 11': 'subjectInformationAccess'
+};
+
+
+// CertificateList  ::=  SEQUENCE  {
+//      tbsCertList          TBSCertList,
+//      signatureAlgorithm   AlgorithmIdentifier,
+//      signature            BIT STRING  }
+var CertificateList = asn1.define('CertificateList', function() {
+  this.seq().obj(
+    this.key('tbsCertList').use(TBSCertList),
+    this.key('signatureAlgorithm').use(AlgorithmIdentifier),
+    this.key('signature').bitstr()
+  );
+});
+rfc5280.CerficateList = CertificateList;
+
+// AlgorithmIdentifier  ::=  SEQUENCE  {
+//      algorithm               OBJECT IDENTIFIER,
+//      parameters              ANY DEFINED BY algorithm OPTIONAL  }
+var AlgorithmIdentifier = asn1.define('AlgorithmIdentifier', function() {
```

Comment:
Could it possibly reuse some declarations from RFC 3280? As far as I get it, it should be possible.